src/pages/o/[orgId]/events/[eventId].tsx

```diff
@@ -73,38 +86,56 @@ export default function OrgEventPage(props : OrgEventPageProps) : JSX.Element {
             </Flex>
             <Flex>
                 <Text marginEnd="size-50">
-                    Start:
-                </Text>
-                <Text data-test="start-time">
-                    { eventQueryData?.start_time }
+                    <Msg id="pages.orgEvent.details.start"/>:
                 </Text>
+                { eventQueryData ? (
+                    <Text data-test="start-time">
+                        <FormattedDate
+                            day="2-digit"
+                            month="long"
+                            value={ Date.parse(eventQueryData.start_time) }
+                        />
+                        , <FormattedTime
+                            value={ Date.parse(eventQueryData.start_time) }
+                        />
+                    </Text>
+                ) : <Msg id="pages.orgEvent.details.org"/> }
```

Comment:
This should not be conditional.

If the event could not be retrieved, we should not reach this point (check for falsy before returning, and return `null` instead).

If the event could be retrieved, we should store it in a variable called `event` of type `ZetkinEvent` and just use `event.start_time` et c here, which will always exist per the contract of the Zetkin API (an action can never be without a start_time).